WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
No, it is completely inaccurate, because it didn't happen. There were some events in late terms on patriarch level, but that is not represented through these events.